src/platform/mhv/downtime/tests/components/MHVDown.unit.spec.jsx

```diff
@@ -0,0 +1,51 @@
+import React from 'react';
+import { render } from '@testing-library/react';
+
+import MHVDown from '../../components/MHVDown';
+
+const endString = 'END DATE';
+const startString = 'START DATE';
+const timeInterval = 'TIME INTERVAL';
+
+describe('MHVDown', () => {
+  it('renders app title when provided', () => {
+    const props = {
+      appTitle: 'APPLICATION',
+      endString,
+      startString,
+      timeInterval,
+    };
+
+    const { getByText } = render(<MHVDown {...props} />);
+    // Use function because testing-library has trouble with text that spans multiple lines (newlines)?!?
+    getByText((content, _) => {
+      return (
+        content.includes(timeInterval) &&
+        content.includes('you may have trouble using') &&
+        content.includes('APPLICATION')
+      );
+    });
+    getByText(startString);
+    getByText(endString);
```

Comment:
Here are a couple of ways to assert rendered text in your component that can handle newlines.

```
    // https://stackoverflow.com/a/70591621
    getByText(/you may have trouble using application/i);

    // https://stackoverflow.com/a/60865407
    getByText('you may have trouble using APPLICATION', { exact: false });
```

Reply:
Sorry, I should have indicated that I had written these with regexes first, but they didn't work. I also tried a few other ways including a customized `Normalizer`. It was weird!

Reply:
Retrying these...

Reply:
Okay, not sure what was giving me problems last week. Updated to use regexes.